css-page-3/lc-doc-20040215.htm

```diff
@@ -1,9377 +1,9407 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

```

Comment:
Woah, what happened with this file?

Reply:
Not sure if there was some encoding issue, but I can always remove it given the age

Reply:
yeah, we should just, uh, not touch that file i guess.  It's a generated file anyway.

Reply:
Rebased it out